src/main/resources/BibleNames_ru.properties

```diff
@@ -150,6 +150,8 @@ John.Short=Ин
 John.Alt=ин,инн
 Acts.Full=Деяния
 Acts.Short=Деян
+Did.Full=Дидахе
```

Comment:
translations are made via Transifex so this will be overwritten. See https://github.com/AndBible/and-bible/wiki/Translating-User-Interface

Reply:
Does that mean the code merge not accepted?

Reply:
No, it's okay, just saying. But translations need to be done also in transifex.

Reply:
So, I think I properly introduced LXXLCV versification in the jsword form, but the system does fail gradle tests.

Reply:
What you could do is to try to build AndBible with your version and try it out with your m odule.

Reply:
> > Did you manage to test this?
> 
> gradle throws thousand of TEST FAIL errors. I thought it is my clone repo errors, but when I ran `./gradlew test `on your repo, same errors appeared.

It looks like CI has been successfull, but not in your branch. Though I have noticed that JSword CI sometimes gives false negatives. https://github.com/AndBible/jsword/actions/runs/4090858645/jobs/7054692044

Reply:
> > > Did you manage to test this?
> > 
> > 
> > gradle throws thousand of TEST FAIL errors. I thought it is my clone repo errors, but when I ran `./gradlew test `on your repo, same errors appeared.
> 
> It looks like CI has been successfull, but not in your branch. Though I have noticed that JSword CI sometimes gives false negatives. https://github.com/AndBible/jsword/actions/runs/4090858645/jobs/7054692044

What command should I run to test my edits besides running `./gradlew test`?